src/Symfony/Contracts/phpunit.xml.dist

```diff
@@ -15,6 +15,7 @@
     <testsuites>
         <testsuite name="Symfony Contracts Test Suite">
             <directory>./Tests/</directory>
+            <directory>./Translation/Test/</directory>
```

Comment:
shouldnt we move all tests to `src/.../Tests/` (note plural).. what happened? :sweat_smile: that's what all components do.

Reply:
reusable tests are in `Test`, tests themselves in `Tests`
TranslatorTest is used in the Translation component.